harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/AbstractParams.java

```diff
@@ -277,7 +277,7 @@ public void update(Element node) throws BadInputEx {
 
         getTrigger();
 
-        setImportXslt(Util.getParam(content, "importxslt", getImportXslt()));
+        setImportXslt(Util.getParam(content, "importxslt", "none"));
```

Comment:
In the CSW code it's used `""`, while here it's used `"none"`, to confirm that the difference is correct?

Reply:
In the `create` of AbstractParam it is set to `none` https://github.com/geonetwork/core-geonetwork/blob/master/harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/AbstractParams.java#L200